<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Filter Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Gaussian</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unsharp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Median</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Convolution (user kernel)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="kernelSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Kernel settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="sizeLabel">
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sizeBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sigmaLabel">
        <property name="text">
         <string>Sigma:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="sigmaBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FilterDialog</receiver>
   <slot>changeType(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeType(int)</slot>
 </slots>
</ui>
